libfabric.spec.in

```diff
@@ -1,11 +1,11 @@
 Name: libfabric
-Version: @VERSION@
+Version: @RPM_VERSION@
 Release: 1%{?dist}
 Summary: User-space RDMA Fabric Interfaces
 Group: System Environment/Libraries
 License: GPLv2 or BSD
 Url: http://www.github.com/ofiwg/libfabric
-Source: http://www.openfabrics.org/downloads/fabrics/%{name}-%{version}.tar.bz2
+Source: http://www.openfabrics.org/downloads/fabrics/%{name}-@PACKAGE_VERSION@.tar.bz2
```

Comment:
Only the Version: substitution needs to happen, all these can remain using %{version}